src/features/CourtCaseDetails/Sidebar/PncDetails/PncCourtCaseAccordion.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from "react"
+import type { PncCourtCase } from "@moj-bichard7-developers/bichard7-next-core/core/types/PncQueryResult"
+import {
+  CCR,
+  CourtCaseHeaderContainer,
+  CrimeOffenceReference,
+  CourtCase,
+  Offence,
+  CourtCaseHeader,
+  ChevronContainer
+} from "./PncDetails.styles"
+import PncOffenceDetails from "./PncOffenceDetails"
+import Disposal from "./Disposal"
+
+interface PncCourtCaseAccordionProps {
+  index: number
+  pncCourtCase: PncCourtCase
+}
+
+const PncCourtCaseAccordion = ({
+  index,
+  pncCourtCase: { courtCaseReference, crimeOffenceReference, offences }
+}: PncCourtCaseAccordionProps) => {
+  const [open, setOpen] = useState<boolean>(index === 0 ? true : false)
+  const handleClick = () => {
+    setOpen(!open)
+  }
+
+  const chevronPosition = open ? "govuk-accordion-nav__chevron--up" : "govuk-accordion-nav__chevron--down"
+
+  return (
+    <CourtCase key={courtCaseReference}>
+      <CourtCaseHeaderContainer className="courtcase-toggle" onClick={handleClick}>
+        <CourtCaseHeader>
+          <CCR className="govuk-heading-m">{courtCaseReference}</CCR>
+          <CrimeOffenceReference>
+            <div className={"heading"}>{"Crime Offence Reference"}</div>
+            <div id={"crime-offence-reference"}>{crimeOffenceReference || "-"}</div>
+          </CrimeOffenceReference>
+        </CourtCaseHeader>
+
+        <ChevronContainer>
+          <span className={`govuk-accordion-nav__chevron ${chevronPosition} chevron`}></span>
+        </ChevronContainer>
+      </CourtCaseHeaderContainer>
```

Comment:
Looking at `src/components/Accordion.tsx`, we should have some `aria` attributes for accessibility.